web/src/lib/utils.ts

```diff
@@ -25,6 +25,7 @@ import { mdiCogRefreshOutline, mdiDatabaseRefreshOutline, mdiHeadSyncOutline, md
 import { sortBy } from 'lodash-es';
 import { init, register, t } from 'svelte-i18n';
 import { derived, get } from 'svelte/store';
+import { ModuleKind, transpileModule } from 'typescript';
```

Comment:
This file is used in other places so we don't want to be importing from typescript here, because that makes it a runtime dependency.